docker_daemon/check.py

```diff
@@ -16,11 +16,9 @@
 from utils.dockerutil import (DockerUtil,
                               MountException,
                               BogusPIDException,
-                              SWARM_SVC_LABEL,
-                              RANCHER_CONTAINER_NAME,
-                              RANCHER_SVC_NAME,
-                              RANCHER_STACK_NAME)
+                              SWARM_SVC_LABEL)
 from utils.kubernetes import KubeUtil
+from utils.rancher import RancherUtil
```

Comment:
will need to move to L24 after the directory move in https://github.com/DataDog/dd-agent/pull/3366